Client/Frontend/Fakespot/Views/AdjustRatingView.swift

```diff
@@ -0,0 +1,146 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import ComponentLibrary
+
+struct AdjustRatingViewModel {
+    let title: String
+    let description: String
+    let titleA11yId: String
+    let cardA11yId: String
+    let descriptionA11yId: String
+    let rating: Double
+}
+
+class AdjustRatingView: UIView, Notifiable, ThemeApplicable {
+    private enum UX {
+        static let titleFontSize: CGFloat = 17
+        static let descriptionFontSize: CGFloat = 13
+        static let margins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
+        static let hStackSpacing: CGFloat = 4
+        static let vStackSpacing: CGFloat = 8
+        static let starSize: CGFloat = 24
+        static let starMaxSize: CGFloat = 42
+    }
+
+    private lazy var cardContainer: ShadowCardView = .build()
+    private lazy var contentView: UIView = .build()
+
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.titleFontSize,
+                                                            weight: .medium)
+        label.numberOfLines = 0
+    }
+
+    private lazy var descriptionLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.descriptionFontSize,
+                                                            weight: .regular)
+        label.numberOfLines = 0
+    }
+
+    private lazy var starRatingView: StarRatingView = .build()
+
+    private lazy var hStackView: UIStackView = .build { stackView in
+        stackView.axis = .horizontal
+        stackView.spacing = UX.hStackSpacing
+    }
+
+    private lazy var vStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.vStackSpacing
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UX.margins
+    }
+
+    private lazy var spacer: UIView = .build()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupNotifications(forObserver: self,
+                           observing: [.DynamicFontChanged])
+        setupLayout()
+    }
+
+    func configure(_ viewModel: AdjustRatingViewModel) {
+        titleLabel.text = viewModel.title
+        titleLabel.accessibilityIdentifier = viewModel.titleA11yId
+
+        descriptionLabel.text = viewModel.description
+        descriptionLabel.accessibilityIdentifier = viewModel.descriptionA11yId
+
+        starRatingView.rating = viewModel.rating
```

Comment:
Just noticed that the `starRatingView` has no a11y id nor a11y label. Like this it doesn't convey any useful information to a voice over user. Could you please add both? Please check with Emily for an appropriate a11y label.

Reply:
Okay. @ewachowiak what do you think the voiceover should read when it gets to the stars rating?

Reply:
Adding Emily's response: "looks like desktop addresses it [like this.](https://www.figma.com/file/Wu4kaHSfKSQnzkym7qek3R/Shopping-Experience-MVP?type=design&node-id=35-10617&mode=design&t=CaBLfUOWZQCSRNcq-4) (See [slack thread ](https://mozilla.slack.com/archives/C4E0W8B8E/p1685972015829989)for reference)"